src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -303,13 +304,9 @@ protected override void Shutdown()
 
             await requestContext.SendResult(null);
 
-            // If no script is being launched, execute an empty script to
-            // cause the prompt string to be evaluated and displayed
-            if (string.IsNullOrEmpty(this.scriptToLaunch))
-            {
-                await this.editorSession.PowerShellContext.ExecuteScriptString(
-                    "", false, true);
-            }
```

Comment:
The prompt has already been written by this point, so this line is totally pointless now.  It was necessary before the integrated console came along because a prompt was never displayed at first when the debugger starts, the assumption was that we were launching a script.  For an "interactive session" I needed to invoke a prompt to display, so this was a hacky way to do that :)